bears.js

```diff
@@ -0,0 +1,152 @@
+const express = require('express');
+const db = require('./data/db');
+
+const server = express.Router();
+
+server.post('/', async (req, res) => {
+
+  const { name } = req.body;
+
+  if (!name) {
+
+    res.status(400).json({message: 'INVALID REQUEST: Body requires name property!'});
+    return;
+
+  }
+
+  try {
+
+    const id = await db.insert({ name }).into('bears');
+    const newObj = await db.select().from('bears').where('id', id[0]);
```

Comment:
The same applies for these in regards to array destructuring:
```javascript
const [ id ] = await db.insert({ name }).into('bears');
const [ newObj ] = await db.select().from('bears').where({ id });
res.status(201).json(newObj);
```

Reply:
Note that anything destructured that isn't there (i.e. destructuring the first item when there are no items) will lead to whichever specific thing(s) being undefined.